sspl_test/rabbitmq/rabbitmq_ingress_processor_tests.py

```diff
@@ -231,6 +236,15 @@ def _configure_exchange(self):
                 self._routing_key,
                 self._queue_name
             )
+            self.cluster_id = self._conf_reader._get_value_with_default(
+                self.SYSTEM_INFORMATION_KEY, self.CLUSTER_ID_KEY, '')
+            # Decrypt RabbitMQ Password
+            decryption_key = encryptor.gen_key(self.cluster_id, ServiceTypes.RABBITMQ.value)
+            try:
+                self._password = encryptor.decrypt(decryption_key, self._password.encode('ascii'), "RabbitMQingressProcessor")
+            except Exception as e:
+                self._password = "sspl4ever"

```

Comment:
This would never work :) It is not a default password anymore. Just raise the exception back. Then we would immediately know something went wrong.